templates/default.json

```diff
@@ -68,6 +82,8 @@
         "<rootDir>/.next/",
         "<rootDir>/node_modules/",
         "<rootDir>/coverage/",
+        "<rootDir>/src/utils/apollo.jsx",
```

Comment:
I'd like to look at what we can do to get passing tests on these new files instead of excluding them from test coverage.